modules/gcp-gke/node_pool_tags.tf

```diff
@@ -0,0 +1,18 @@
+###
+# This file contains no business logic.
+# The solely goal is to provide a way to read out the tags of the node pool, which include the ones we apply with this component + the ones Generated by Google.
+###
+
+locals {
+  # includes the by GCP-managed tags
+  primary_node_pool_all_tags = sort(data.google_compute_instance.primary_node_pool.tags)
```

Comment:
Why are we sorting these?

Reply:
Alphabetically ordered. Not required but definitely useful when comparing outputs visually.

Example:
If one run gives out 
```
["a", "b"]
```
And another run gives out
```
["b", "a"]
```

It'd be a bit more readable to have them sorted out.
Can remove the sort if necessary, not a big deal. Let me know.